tests.yaml

```diff
@@ -38,15 +38,15 @@ tests:
     values: '32768+0x61'
   alert_rule_test:
   - eval_time: 1h
-    alertname: KubePersistentVolumeFullInFourDays
+    alertname: KubePersistentVolumeFillingUp
   - eval_time: 61m
-    alertname: KubePersistentVolumeFullInFourDays
+    alertname: KubePersistentVolumeFillingUp
     exp_alerts:
     - exp_labels:
         job: kubelet
         namespace: monitoring
         persistentvolumeclaim: somepvc
-        severity: critical
+        severity: warning
       exp_annotations:
         message: 'Based on recent sampling, the PersistentVolume claimed by somepvc in Namespace monitoring is expected to fill up within four days. Currently 1.263% is available.'
         runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubepersistentvolumefullinfourdays
```

Comment:
I think the problem with both of the alerts being named the same is link to the runbook as well @brancz thoughts?

Reply:
They describe the same problem, so this seems fine.